src/main/java/com/khomishchak/ws/services/security/JwtServiceImpl.java

```diff
@@ -88,6 +85,23 @@ public String generateToken(Map<String, String> extraClaims, UserDetails userDet
         }
     }
 
+    private UserDetailsImpl getUserDetailsImpl(UserDetails userDetails) {
+        if(!(userDetails instanceof UserDetailsImpl)) {
+            throw new IllegalArgumentException("Expected type was UserDetailsImpl");
+        }
+
+        UserDetailsImpl userDetailsimpl = (UserDetailsImpl) userDetails;
+
+        String username = userDetailsimpl.getUsername();
+        Long userId = userDetailsimpl.getUserId();
+
+        if(username == null || username.isBlank() || userId == null) {
```

Comment:
```suggestion
        if(!StringUtils.hasText(username) || userId == null) {
```
https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/util/StringUtils.html#hasText(java.lang.CharSequence)